source/reference/code-blocks.txt

```diff
@@ -0,0 +1,467 @@
+         :language: javascript
+
+         [
+             { title: 'The Life Aquatic with Steve Zissou', ... }
+         ]
+
+The previous ``io-code-block`` directive creates the following output:
+
+.. io-code-block::
+   :source: https://github.com/mongodb/docs-node/blob/0cf95d4eac536ed0ae1a2085948dd98f24ac48af/source/quick-reference.txt#L473
+   :copyable: true
+
+   .. input::
+      :language: javascript
+         
+      coll.find({ $text: { $search: 'zissou' } });
+
+   .. output::
+      :language: javascript
+
+      [
+          { title: 'The Life Aquatic with Steve Zissou', ... }
+      ]
+
+Literal Includes
+----------------
+
+The ``literalinclude`` directive includes and formats a code example from a file that you
+specify.
+This directive allows you to maintain the code in a separate single file and display the
+same code example in multiple places.
+
+.. note::  include vs. literalinclude
+
+    Use the ``literalinclude`` directive to include a code file in the docs.
+    Use the ``include`` directive to include an ``rst`` file in the docs.
+
+Syntax
+~~~~~~
+
+.. code-block::
+
+   .. literalinclude:: <file path>
+      :language: <language>
+      :copyable: <boolean>
+      :start-after: <line of code>
+      :end-before: <line of code>
+      :linenos:
+      :lineno-start: <line number>
+      :emphasize-lines: <line numbers>
+      :caption: <caption>
+      :dedent: <number>
+
+Options
+~~~~~~~
+
+.. list-table::
+
+   * - ``<file path>``
+     - **Required.** Path to the file in the repository that contains the code to show.
+
+       **Default:** None
+
+   * - ``:language: <language>``
+     - Language to use for syntax highlighting. 
+       
+       For a complete list of supported languages, see the
+       `leafygreen-ui GitHub repository. <https://github.com/mongodb/leafygreen-ui/tree/main/packages/code#properties>`__
+
+       **Default:** None
+ 
+   * - ``:copyable: <boolean>``
+     - Option that indicates whether to show the :guilabel:`copy icon`.
+
+       **Default:** ``true``
+
+   * - ``:start-after: <line of code>``
+     - Line of code before the first line to include in the sample.
+
+       You can use a comment to mark the point where the ``literalinclude`` directive
+       starts including code. If you do, omit the comment marker (``#`` or ``//``) from
+       the attribute value.
+       
+       By using comments, you can unambiguously
+       specify the start of the code example, regardless of how many examples you're
+       including from the file. You also won't need to update the directive if the
+       extracted code changes.
+
+       **Default:** Beginning of file
+   
+   * - ``:end-before: <line of code>``
+     - Line of code after the last line to include in the sample.
+
+       You can use a comment to mark the point where the ``literalinclude`` directive
+       stops including code. If you do, omit the comment marker (``#`` or ``//``) from
+       the attribute value.
```

Comment:
Suggestion:

I think based on this statement, it would be helpful to omit the "#" in the final example on the page so that conforms to the guidance. Since omitting is optional, maybe rewording this as "If you do, you can omit..." is the only change needed.

Reply:
good thought